src/checkout/views/Review/View.tsx

```diff
@@ -41,15 +36,26 @@ const completeCheckout = (
     });
     clearCheckout();
     clearCart();
-    show(OverlayType.message, null, {
-      status: "success",
-      title: "Your order was placed"
-    });
+    alert.show(
+      { title: "Your order was placed" },
+      {
+        type: "success"
+      }
+    );
+    alert.show(
+      { title: "Your order was placed again" },
```

Comment:
I wonder if we could come up with a better sentence than this one? It is sounding really weird. But I have no good suggestion in mind.

Reply:
"Your order was placed again"  will be removed, it was added only to show how it looks with multiple notifications